packages/battery/battery_linux/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
```

Comment:
Please sync with master and re-copy this LICENSE file from one of the files in the repository; I just standardized the formatting of all of them (it's not yet enforced by any automated checks).

Reply:
Done